cdm/src/main/java/thredds/crawlabledataset/CrawlableDatasetAmazonS3.java

```diff
@@ -3,12 +3,17 @@
 import com.amazonaws.services.s3.AmazonS3Client;
 import com.amazonaws.services.s3.model.*;
 import net.sf.ehcache.*;
+import net.sf.ehcache.constructs.blocking.SelfPopulatingCache;
+import net.sf.ehcache.constructs.blocking.CacheEntryFactory;
 import net.sf.ehcache.event.CacheEventListener;
-import org.apache.commons.io.IOUtils;
+
+import thredds.crawlabledataset.utils.S3Helper;
+import thredds.crawlabledataset.utils.S3ListingCacheEntryFactory;
+import thredds.crawlabledataset.utils.S3ObjectCacheEntryFactory;
```

Comment:
How about s3utils?  utils makes it sound like the package contains more general utilities